src/partials/partial.erb.html

```diff
@@ -1,17 +1,19 @@
-<div data-key="lmn-book" data-name="<%= customisation.name %>"
-  data-first-book-name="Adele" data-first-book-gender="boy" data-first-book-locale="en-GB"
-  data-show-picker="true"
-  data-show-overlay="true"
-  data-gender="<%= customisation.gender %>" data-locale="<%= customisation.locale %>"
-  data-character-selection='[{"letter":"A","character":"Aardvark"},{"letter":"N","character":"Narwhal"},{"letter":"D","character":"Dragon"},{"letter":"R","character":"Robot"},{"letter":"E","character":"Elk"},{"letter":"W","character":"Wizard"}]'
-  class="lmn-book monkey" id="monkey">
-  <div class="color-bg-white lg-pad aligned-center loader-img">
-    <%= image_tag('loading-32x32.gif', { width: 32, class: 'padded-bottom' }) %>
-    <h6 class="not-strong color-white"><%= t("loading") %></h6>
+<div style="margin: 0 auto; max-width: 1004px;">
```

Comment:
IT's used, but literally just to house the monkey wrapper in the dummy preview page we load, which is already like 18 months out of date, so not too concerned about this.